File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    var answer = Array(52).fill(0);
    let al = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";

    for (let i = 0; i < my_string.length; i++) {
        answer[al.indexOf(my_string[i])]++
    }
    return answer;
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = [0] * 52
    al = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    for char in my_string:
        index = al.index(char)
        answer[index] += 1
    return answer